Split user stories for Flatpak (CQA) (#5006)
theforeman/foreman-documentation: guides/common/modules/con_installing-flatpak-applications-on-hosts.adoc

:_mod-docs-content-type: CONCEPT

[id="installing-flatpak-applications-on-hosts"]
= Installing Flatpak applications on hosts

[role="_abstract"]

> After you mirror Flatpak repositories in {Project}, you can install Flatpak applications on hosts from {ProjectServer} or {SmartProxyServer}.

Reviewer: I suggest to use "synchronize" instead of "mirror"; and "to" instead of "in Project".

> Hosts use the command line to add a Flatpak remote, authenticate to the container registry, and install applications.

Reviewer: IMO using the CLI here is not the main point. What about this:

```suggestion
On your hosts, add a Flatpak remote, authenticate to the container registry, and install applications.
```

Alternative: `Your hosts will add ...`.
